mbtools/html_to_json.py

```diff
@@ -0,0 +1,40 @@
+import argparse
+import glob
+import json
+from pathlib import Path
+
+
+def create_json_content(uuid, content, variant="main"):
+    json_content = {"id": uuid, "content":
+                    [{"variant": variant, "html": content}]}
+    return json.dumps(json_content, indent=4)
+
+
+def html_to_json(html_directory, json_directory):
+    file_list = []
+    for file_name in glob.iglob(f'{html_directory}/*.html'):
+        with open(f'{file_name}') as f:
+            file_list.append({"name": Path(file_name).stem,
+                              "content": f.read()})
+
+    for file in file_list:
+        new_file = open(f'{json_directory}/{file["name"]}.json', "w")
+        new_file.write(create_json_content(file["name"], file["content"]))
+        new_file.close()
```

Comment:
This probably isn't a practical issue in our use cases, but here you're creating an in-memory array with all of the HTML content which you subsequently iterate on. As an alternative implementation which would not use O(N) memory resources, you could write the JSON after reading the corresponding HTML in a single loop.